Dockerfile.freyja_apps

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+# syntax=docker/dockerfile:1
```

Comment:
Should this be left in?  What purpose does it serve?

Reply:
It was added when I first created the Dockerfile via a `docker init` and then has been cloned into other dockerfiles that have been created. It appears that it is the stable channel of the dockerfile frontend. https://docs.docker.com/build/dockerfile/frontend/

Reply:
Is it needed?

Reply:
Checking by removing it and re-building the dockerfile

Reply:
Removed